ui/components/atoms/A006 - Typography/stories.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withNotes } from '@storybook/addon-notes'
+
+import Typography from '.'
+
+// import Button from '.'
+import notes from './readme.md'
+
+storiesOf('Atoms|A006 - Typography', module)
+.addDecorator(withNotes)
+.add('Display 4', () => (
+    <Typography title={'display4'} color={'primary'} variant={'display4'}/>
```

Comment:
add spacing in title text between "display" and number

Also Can you captilise all the titles in this file as well. I know i start with not. but it will look nicer done caps